msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -38,7 +38,7 @@ The code snippets in the [samples](#samples) section are to create a one-to-one
 
 ## Get the user ID, team ID, or channel ID
 
-To create a new conversation or a conversation thread in a channel, you must have the correct ID. You can receive or retrieve this ID using any of the following ways:
+You can create a new conversation or a conversation thread in a channel and you must have the correct ID. You can receive or retrieve this ID using any of the following ways:
```

Comment:
Shall we change it to "a new conversation with a user"?

Reply:
updated the doc as suggested.